DoePaAdminViewModel/ManageAuftraegeViewModel.cs

```diff
@@ -37,6 +37,14 @@ public ObservableCollection<Abrechnungseinheit> Abrechnungseinheiten
             set => SetProperty(ref _abrechnungseinheiten, value, true);
         }
 
+        private ObservableCollection<Geschaeftsjahr> _geschaeftsjahre = new();
+
+        public ObservableCollection<Geschaeftsjahr> Geschaeftsjahre
```

Comment:
Please do not bind to this property. Rollback this change and bind the combobox directly to the GeschaeftsjahreViewmodel as we do it in ManageMitarbeiterWindow with the cost centers